auth-for-mcp/fastmcp-mcp-customtokenexchange-js/README.md

```diff
@@ -33,7 +33,7 @@ MCP_AUTH0_CLIENT_SECRET=your-client-secret
 MCP_AUTH0_SUBJECT_TOKEN_TYPE=urn:fastmcp:mcp
 
 # Scopes to request
-MCP_AUTH0_EXCHANGE_SCOPE=read:tasks
+MCP_AUTH0_EXCHANGE_SCOPE=read:private
```

Comment:
Updating this to match with: 
https://github.com/auth0-samples/auth0-ai-samples/blob/9097bd6f01ca2a5c823c827951307b52e941a564/auth-for-mcp/fastmcp-mcp-customtokenexchange-js/src/api/index.ts#L18